pkg/types/compactor.go

```diff
@@ -0,0 +1,53 @@
+package types
+
+import (
+	"fmt"
+	"io/ioutil"
+	"time"
+)
+
+// CompactionResult holds the compaction details after a compaction is done
+type CompactionResult struct {
+	SnapshotName           string
+	LastCompactionDuration time.Duration
+}
+
+// CompactionConfig holds the configuration data for compaction
+type CompactionConfig struct {
+	*RestorationConfig
+}
```

Comment:
Why not use `RestorationConfig` directly?

Reply:
Yes, I could do that. But compaction command using Restorationconfig would be difficult to understand while reading the code. So, I did that.

Reply:
I removed `CompactionConfig` all together